styles/buttons/_mixins.scss

```diff
@@ -1,3 +1,4 @@
 @mixin tap-highlight-color($color) {
-    tap-highlight-color: $color;
+    // Prefix is needed because autoprefix will not prefix non standard css
+    -webkit-tap-highlight-color: $color;
```

Comment:
thats the point. there is no tap-highlight-color, its not part of the spec. Only web kit supports it.